ddterm/app/application.js

```diff
@@ -350,6 +350,8 @@ class Application extends Gtk.Application {
             if (!(ex instanceof GLib.Error &&
                 ex.matches(GLib.file_error_quark(), GLib.FileError.NOENT)))
                 logError(ex, "Can't restore session");
+
+            GLib.unlink(this.session_file_path);
```

Comment:
This line will be executed even if the error is `GLib.FileError.NOENT`. Or any other I/O error. I don't think removing the file here is a good idea.

Reply:
removed this